sfxclient/sfxclient_test.go

```diff
@@ -393,13 +402,15 @@ func BenchmarkScheduler_ReportOnce_With_Debug(b *testing.B) {
 	var handleErrors []error
 	var handleErrRet error
 	s := &Scheduler{
+		CollectorCallbackManager: &CollectorCallbackManager{
```

Comment:
my only real issue is that we'll have to make these same changes all through every use of the code everywhere and while the the collector vs scheduler is a fine metaphor, does it buy us anything?

Reply:
This should be a no-op if we use this package's NewScheduler() constructor downstream.  But if we don't, then it would require updates where we directly create Schedulers.